### PR TITLE
[docs] Add information and command to access Deckhouse controller in HA mode 

### PR DESCRIPTION
**Pull request by haosb**
## Description

This PR will add information and command to access Deckhouse controller in HA mode.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Add information and command to access Deckhouse controller in HA mode.
impact_level: default

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @haosb. I've started the workflow for label 'deploy/web/stage'...

**Review by miklezzzz — APPROVED** [review]
the command and its documentation in general are fine, but I can't help wondering if we should go this way instead of providing a more elegant way of working with replicated Deckhouse instances in the first place.

**Review by kirkonru — CHANGES_REQUESTED** [review]

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 1]
```diff
@@ -891,6 +891,16 @@ kubectl -n d8-system exec -ti deploy/deckhouse -c deckhouse -- deckhouse-control
 
    Вывод команды должен быть пуст.
 
+### Как получить доступ к контроллеру Deckhouse в multi-master кластере?
+
+В кластерах с несколькими master-узлами Deckhouse запускается в режиме высокой доступности. Поэтому для привычного доступа к контроллеру Deckhouse нужно использовать команду:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:896]:
```suggestion
В кластерах с несколькими master-узлами Deckhouse запускается в режиме высокой доступности (в нескольких экземплярах). Для доступа к активному контроллеру Deckhouse можно использовать следующую команду (на примере команды `deckhouse-controller queue list`):
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 2]
```diff
@@ -891,6 +891,16 @@ kubectl -n d8-system exec -ti deploy/deckhouse -c deckhouse -- deckhouse-control
...
+
+```shell
+kubectl -n d8-system exec -it $(kubectl -n d8-system get leases.coordination.k8s.io deckhouse-leader-election -o jsonpath='{.spec.holderIdentity}' | awk -F'.' '{ print $1 }') -c deckhouse -- deckhouse-controller queue list
+```
+
+> В примере выше указана команда `queue list` для просмотра очереди Deckhouse. Её можно заменить на любую привычную вам команду.
+
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:903]:
```suggestion

```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 3]
```diff
@@ -889,6 +889,16 @@ To switch Deckhouse Community Edition to Enterprise Edition, follow these steps:
 
    The output of the command should be empty.
 
+### How do I get access to Deckhouse controller in multimaster cluster?
+
+In clusters with multiple master nodes, Deckhouse runs in high availability mode. This is why you have to use the following command to access the Deckhouse controller:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:894]:
```suggestion
In clusters with multiple master nodes Deckhouse runs in high availability mode (in several instances). To access the active Deckhouse controller, you can use the following command (as an example of the command `deckhouse-controller queue list`):
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 4]
```diff
@@ -889,6 +889,16 @@ To switch Deckhouse Community Edition to Enterprise Edition, follow these steps:
...
+
+```shell
+kubectl -n d8-system exec -it $(kubectl -n d8-system get leases.coordination.k8s.io deckhouse-leader-election -o jsonpath='{.spec.holderIdentity}' | awk -F'.' '{ print $1 }') -c deckhouse -- deckhouse-controller queue list
+```
+
+> In the example above, the `queue list` command lists the Deckhouse queue. It is used for illustrative purposes only. You can replace it with any command you need.
+
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:901]:
```suggestion

```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8843](https://github.com/deckhouse/deckhouse/pull/8843) to the branch [release-1.61](https://github.com/deckhouse/deckhouse/tree/release-1.61) successful!

**Pull request merged**